rosco/controller/src/DISCON.F90

```diff
@@ -100,6 +100,7 @@ SUBROUTINE DISCON(avrSWAP, aviFAIL, accINFILE, avcOUTNAME, avcMSG) BIND (C, NAME
         CALL UpdateZeroMQ(LocalVar, CntrPar, ErrVar)
     ENDIF
     
+    CALL Shutdown(LocalVar, CntrPar, objInst,ErrVar)

```

Comment:
Should there be an `IF (SD_Mode > 0)` here?

Reply:
Yes, that makes sense. Making the change.

Reply:
resolved in 96f1f545